app/src/js/components/SortableTable/SortableTable.js

```diff
@@ -144,7 +159,7 @@ const SortableTable = ({
                   {headerGroup.headers.map(column => {
                     return (
                       <div {...column.getHeaderProps()} className='th'>
-                        <div {...column.getSortByToggleProps()} className={`${column.canSort ? 'table__sort' : ''}`}>
+                        <div {...column.getSortByToggleProps()} className={`${column.canSort ? 'table__sort' : ''}${column.isSortedDesc === true ? '--desc' : (column.isSortedDesc === false ? '--asc' : '')}`}>
```

Comment:
It's not possible for column to look like `{canSort: false, isSortedDesc: true, ...}`? I would guess not.

Reply:
Ah interesting point. Correct, that would not be possible, though it might make sense to more clearly encapsulate that logic here

Not sure if this is more or less confusing 😂 

```
className={`${column.canSort ? (`table-sort${column.isSortDesc === true ? '--desc' : (column.isSortedDesc === false ? '--asc' : '')}`) : ''}`}
```

or could do something like this

```
let columnClassName = '';
if (column.canSort) {
  columnClassName = `table-sort${column.isSortedDesc === true ? '--desc' : (column.isSortedDesc === false ? '--asc' : '')}`;
}
```

Reply:
It's fine as it, but I like the last one best. your call.

Reply:
Yep, agreed, changed it to the last one